setup.py

```diff
@@ -23,6 +23,8 @@
     python_requires='~=3.7',
     install_requires=[
         "pandas",
```

Comment:
can we somehow make pandas an optional dependency? is there a way to do that in python? as in: package works completely without pandas, but if you want to use the pandas functionality you can install it.

Reply:
Yes, but we need pandas for some checks, so we need to set it as a requirement, unless we rearrange the code.